steps/redis/session.go

```diff
@@ -106,13 +139,16 @@ func (s *Session) ValidateJSONValue(ctx context.Context, key string, props map[s
 
 // ValidateEmptyValue checks if the redis key has not value.
 func (s *Session) ValidateEmptyValue(ctx context.Context, key string) error {
-	_, err := s.Client.Get(context.Background(), key).Result()
+	exists, err := s.Client.Exists(context.Background(), key).Result()
```

Comment:
What's the problem with Get? What's the difference?

Reply:
[Exists][exists] is a function to ensure a key exists in Redis database. On the other hand, [Get][get] is a function to retrieve a string value from redis, and it will return an error if it is used with other types of values such as hash values.

[get]: https://www.tutorialspoint.com/redis/strings_get.htm
[exists]: https://www.tutorialspoint.com/redis/keys_exists.htm